testutils/options.go

```diff
@@ -12,8 +12,16 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-type Option func(*unstructured.Unstructured) error
-type NATSOption func(*nmapiv1alpha1.NATS) error
+var (
+	ErrFailedToConvertStatusToMap   = errors.New("failed to convert status to map[string]interface")
```

Comment:
Might be out of scope: but is there a best practice whether or not this should be started with a capital letter and end with a "."? Maybe this is actually addressed in a different linter?

Reply:
Do you mean the error message? 
Error messages must be started with lower letter. No best practice for end with "."